tasks/taglib.rb

```diff
@@ -0,0 +1,37 @@
+# frozen-string-literal: true
+
+class Taglib
+  class << self
+    def plat
+      ENV['PLATFORM'] || 'i386-mingw32'
+    end
+
+    def version
+      ENV['TAGLIB_VERSION'] || '1.9.1'
+    end
+
+    def dir
+      "taglib-#{version}"
+    end
+
+    def tmp
+      "#{__dir__}/../tmp"
+    end
+
+    def source
+      "#{tmp}/#{dir}"
+    end
+
+    def tmp_arch
+      "#{tmp}/#{plat}"
+    end
+
+    def install_dir
+      "#{tmp_arch}/#{dir}"
+    end
+
+    def library
```

Comment:
I created this `Taglib` class so that `swig.rake` can depend on targets defined in `ext.rake`.